packages/query/src/common/remote-config/remote-config.query.ts

```diff
@@ -0,0 +1,153 @@
+import { createMoney, isUndefined } from '@leather-wallet/utils';
```

Comment:
I believe you will need to update this with the new `runesEnabled` config change.